tests/AppTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Fuzz\ImageResizer\Tests;
+
+use Fuzz\ImageResizer\App;
+use Fuzz\ImageResizer\File;
+use Fuzz\ImageResizer\Image;
+use Imagick;
+use Mockery;
+use Symfony\Component\HttpFoundation\Request;
+
+class AppTest extends ImageResizerTestCase
+{
+	public function getImageFile($height, $width)
+	{
+		return File::createFromBlob(file_get_contents($this->getPlaceholditImage($height, $width)));
+	}
+
+	/**
+	 * @runInSeparateProcess
+	 */
+	public function testItAltersFile()
```

Comment:
Split into `testCropping` and `testCompression`? In my own tests, I tend to segment test cases more granularly, but at least these two functions should exist.